tests/cypress/tests-e2e/featureSelection.cy.js

```diff
@@ -0,0 +1,212 @@
+/// <reference types="cypress" />
+import { FeatureInfoPositions } from '@/store/modules/ui.store'
+
+describe('Testing the feature selection in the URL', () => {
+    function checkFeatures() {
+        cy.readStoreValue('state.features.selectedFeatures').then((features) => {
+            cy.get('@featuresIds').then((featuresIds) => {
```

Comment:
To avoid flakiness might make worse a try to put this the other way around

```javascript
// Read first the fixture that don't require retries
cy.get('@featuresIds').then((featuresIds) => {
 cy.readStoreValue('state.features.selectedFeatures').should((features) => {
  expect(features.length).to.be.equal(featuresIds.length)
  features.forEach((feature) => {
    expect(feature)...
  })
  })
})
```

Reply:
good idea